### PR TITLE
Move SQSWorker to common as Trait

### PR DESCRIPTION
**Pull request by kenoir**
### What is this PR trying to achieve?

Laying the groundwork for standardising our service workers to use SQS only as an event stream - moves SQSWorker to common and makes it into a Trait.

### Who is this change for?

Developers who want to reduce the types of event stream interface to our services.

### Have the following been considered/are they needed?

- [ ] Deployed new versions

### CODE REVIEW

File: ingestor/src/main/scala/uk/ac/wellcome/platform/ingestor/modules/IngestorWorker.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package uk.ac.wellcome.platform.ingestor.modules
+
+import com.amazonaws.services.sqs.model.{Message => AwsSQSMessage}
```
Comment (alexwlchan) [inline ingestor/src/main/scala/uk/ac/wellcome/platform/ingestor/modules/IngestorWorker.scala]: Do you need this import?

Comment (kenoir) [inline ingestor/src/main/scala/uk/ac/wellcome/platform/ingestor/modules/IngestorWorker.scala]: No. Will remove.

**Comment by kenoir** [comment]
Part of https://github.com/wellcometrust/platform-api/pull/553

**Comment by kenoir** [comment]
Part of https://github.com/wellcometrust/platform-api/issues/478

**Pull request merged**